scripts/runcode.jl

```diff
@@ -257,7 +257,7 @@ function chamber(
 end
 
 """
-    chamber(composition::Union{Silicic,Mafic}, end_time::Float64, log_volume_km3_vector::Union{Float64, Vector{Float64}}, InitialConc_H2O_vector::Union{Float64, Vector{Float64}}, InitialConc_CO2_vector::Union{Float64, Vector{Float64}}, log_vfr_vector::Union{Float64, Vector{Float64}}, depth_vector::Union{Float64, Vector{Float64}}, output_dirname::String)
+    arraychamber(composition::Union{Silicic,Mafic}, end_time::Float64, log_volume_km3_vector::Union{Float64,Vector{Float64}}, InitialConc_H2O_vector::Union{Float64,Vector{Float64}}, InitialConc_CO2_vector::Union{Float64,Vector{Float64}}, log_vfr_vector::Union{Float64,Vector{Float64}}, depth_vector::Union{Float64,Vector{Float64}}, output_dirname::String)
```

Comment:
I think the same name can be used and everything should still work! Julia will know which method to call.
```suggestion
    chamber(composition::Union{Silicic,Mafic}, end_time::Float64, log_volume_km3_vector::Union{Float64,Vector{Float64}}, InitialConc_H2O_vector::Union{Float64,Vector{Float64}}, InitialConc_CO2_vector::Union{Float64,Vector{Float64}}, log_vfr_vector::Union{Float64,Vector{Float64}}, depth_vector::Union{Float64,Vector{Float64}}, output_dirname::String)
```